clang/test/CheckedCRewriter/no_casts.c

```diff
@@ -1,4 +1,4 @@
-// RUN: cconv-standalone %s | FileCheck -match-full-lines %s
+// RUN: cconv-standalone %s | count 0
```

Comment:
What's going on with this?

Reply:
This means no conversion should happen. Since there’s not output, the only thing to “check” is that the number of converted lines is 0.